EnglishFilter/sections/specific.txt

```diff
@@ -258,6 +258,10 @@ iscb2015.info##a[target="_blank"] > img:not([class]):not([alt])
 family-fuck.net##.plink
 ||imzog.com/poppy/
 ||audioz.download/templates/Default/img/promo/
+8boobs.com##.sidebar > div.photo > a[target="_parent"] > img[width="230"]
+8boobs.com##div[align="center"] > a[href^="/click/"]
+||8boobs.com/flr.js
+||8boobs.com/bload
```

Comment:
You can make a common rules. Those sites are similar.
https://github.com/AdguardTeam/AdguardFilters/pull/70219